Installing-and-Configuring/RecoveryShell.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: Heads Recovery Shell
+permalink: /RecoveryShell/
+nav_order: 90
+has_children: false
+has_toc: false
+parent: Installing and configuring
+---
+
+Heads Recovery Shell
+====
+
+The recovery shell is a command line where heads and *nix utilities may be accessed.  This document is a work in progress. 
+
+
+Overview
+---
+
+The recovery shell is running a Linux kernel and shell based on busybox.  The environment comes from the initrd image flashed into the BIOS with the Linux kernel.  i.e.  heads
+
+The shell can be used to troubleshoot heads.  The security token paired with heads may be needed to effectively use this environment.
+
+
+Access
+----
+
+The recovery shell may be accessed on boot by holding down 'r' during boot or by choosing the appropriate menu in the heads GUI.  These two different methods of access will result in some different settings.
```

Comment:
typing r frenetically, not simply holding r

Reply:
fixed.